.github/workflows/fossa.yaml

```diff
@@ -0,0 +1,16 @@
+name: FOSSA license check
+on: [pull_request]
+
+# https://github.com/marketplace/actions/fossa-action
+
+jobs:
+  fossa:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: fossa-contrib/fossa-action@v1
+        with:
+          # https://docs.fossa.com/docs/api-reference#push-only-api-token
+          fossa-api-key: 831247a6ab6f0d4f99879e0353b512a5
```

Comment:
is this key statically assigned to us some how ?

Reply:
It's a push-only API key I created.  That's what the docs said to do for the github action

https://github.com/marketplace/actions/fossa-action

Reply:
do u need to plugin somethig to config.yaml for this to be part of the CI ?

Reply:
there is also fossa analyze in this example https://github.com/k8ssandra/cass-operator/blob/master/.github/workflows/license-check.yml

Reply:
This is for github actions.
Here I guess it's using a circleCI orb.